members/hwigwang_lim.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/hwigwang_lim.png" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Hwigwang Lim (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Graduate Student Researcher
+
+<!-- Name at least one research topic amongst this list -->
+**Things I'm excited about right now**: Behavior and role of particles in ionic liquid. Representation of charged molecules and particles.
+
+<!-- Ultimately, we'll use this section to
+     include papers and talks, and contributions
+     But for now put whatever you want -->
+**Cool stuff I've done**: Carving Bucky Badger on a pumkin, watching movies for 8 hours a day, and all the things meaningless and meaningful at the same time.
```

Comment:
### 
```suggestion
**Cool stuff I've done**: Carved Bucky Badger on a pumpkin, watched movies for 8 hours in one day, and many things meaningless and meaningful at the same time.
```

Reply:
Great! I just added a commit for this suggestion.